gammapy/datasets/evaluator.py

```diff
@@ -406,9 +404,9 @@ def parameter_norm_only_changed(self):
         norm_only_changed = False
         idx = self._norm_idx
```

Comment:
Looking at this part here, I wonder if the definition of `_norm_idx` is optimal. It does not rely on the `Parameter.is_norm` property but on the names of the parameters. 
This could be improved since `norm_idx` is `None` if both `amplitude` and `norm` parameters are present.